src/components/indivStats/studentStats/StudentStats.js

```diff
@@ -1,7 +1,7 @@
 import axios from "axios";
 import React, { useEffect, useState } from "react";
 import "../../../components/dashboard/dashboard.scss";
-import { BACKEND_URL, STATS_API } from "../../../constants/constants";
+import * as K from "../../../constants/constants";
```

Comment:
Let's import only the things that are needed as it's easier to using `STATS_API` directly rather than using `K.STATS_API`.

Reply:
I was thinking that it would be better that we can access all the constants by just typing "K."(after which we will get all the variable names to our plate to choose from). Also, it would be easy not to remember specific variables as if their numbers increase, it will become difficult to search and then type them out. What do you think?

Reply:
Thanks @sahil-shubham for the suggestions. I have implemented them. Let me know if anything else is needed.